Resources/public/js/alloyeditor/toolbars/config/block-floating-base.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+/* global CKEDITOR */
+YUI.add('ez-alloyeditor-toolbar-config-block-floating-base', function (Y) {
+    "use strict";
+     /**
+     * Provides base methods to toolbar dedicated to block element (heading,
+     * paragraph, ...).
+     *
+     * @module ez-alloyeditor-toolbar-config-block-floating-base
+     */
+    Y.namespace('eZ.AlloyEditorToolbarConfig');
+
+    var ReactDOM = window.ReactDOM;
+    var FLOATING_TOOLBAR_FIXED_CLASS = 'ae-toolbar-floating-fixed';
+
+    function setPositionFor (block, editor) {
+        /* jshint validthis: true */
```

Comment:
What was the problem for jshint?

Reply:
Refactored in https://github.com/ezsystems/PlatformUIBundle/pull/962/commits/eb11432752657727aa825cb770fceec5a00970a0 so this directive shouldn't be necessary now 😉